script/build.ts

```diff
@@ -55,15 +58,31 @@ updateLicenseDump(err => {
       process.exit(1)
     } else {
       console.log(`Built to ${appPaths}`)
-      process.exit(0)
     }
   })
 })
 
-function packageApp(callback) {
-  const options = {
-    name: distInfo.getExecutableName(),
-    platform: process.platform,
+function packageApp(
+  callback: (error: Error | null, appPaths: string | string[]) => void
+) {
+  // not sure if this is needed anywhere, so I'm just going to inline it here
+  // for now and see what the future brings...
+  const toPackagePlatform = (platform: NodeJS.Platform) => {
```

Comment:
Why do we have a function that takes `platform` as an argument and proceed to not use that argument?

I think this should be `platform === 'win32...'